src/test/java/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String VALID_PATH = "src\\main\\resources\\fileReaderTestFile.csv";
```

Comment:
в тестах краще звертатись до фалів за таким шляхом
```suggestion
    private static final String VALID_PATH = "src\\test\\resources\\fileReaderTestFile.csv";
```